src/style.css

```
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+}

```

```suggestion
* {
  margin: 0;
}
```

Forbidden to set margin in `*` selector